### PR TITLE
ipatests: fix check for AD topology being present

### PR DESCRIPTION
**Pull request by abbra**
Fixes: https://pagure.io/freeipa/issue/9133

Signed-off-by: Alexander Bokovoy <abokovoy@redhat.com>

### CODE REVIEW

**Comment by abbra** [comment]
`test_fips` run succeeded: http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/72cef63c-adb8-11ec-97ef-fa163e04db38/report.html

**Comment by abbra** [comment]
`test_smb` failed due to known issue with bash readline changes. It is being addressed by https://github.com/freeipa/freeipa-pr-ci/pull/455

**Comment by abbra** [comment]
@flo-renaud I think this is ready for review.

File: ipatests/pytest_ipa/integration/tasks.py [hunk 1]
```diff
@@ -570,7 +570,7 @@ def install_client(master, client, extra_args=[], user=None,
 
     args.extend(extra_args)
 
-    if is_fips_enabled(client) and 'ad' in master:
+    if is_fips_enabled(client) and getattr(master, 'ad_domains', False):
```
Comment (flo-renaud) [inline ipatests/pytest_ipa/integration/tasks.py:573]: The ad_domains attribute is defined in master.config, not in master:
```
(Pdb) type(master)
<class 'ipatests.pytest_ipa.integration.host.Host'>
(Pdb) getattr(master, 'ad_domains', False)
False
(Pdb) getattr(master.config, 'ad_domains', False)
[<ipatests.pytest_ipa.integration.config.Domain object at 0x7f852f149a30>]
```

Comment (abbra) [reply ipatests/pytest_ipa/integration/tasks.py]: fixed, thanks! I was confused as there is no explicit `config` being assigned to the host, only to domain. 
I suspect this comes from `ipa-test-task` binary.

**Comment by flo-renaud** [comment]
@abbra thanks for the update, LGTM

**Comment by abbra** [comment]
master:

* 27ab21658bce520fcd559f5a7a2b20d7451c3989 ipatests: fix check for AD topology being present

**Pull request closed without merging**